### ISSUE TITLE
Unable to reproduce the results of ORL dataset. 

### ISSUE THREAD

**Issue by kowshikthopalli** [9310gaurav/DSCN#1]
Hi, When I run your code using AE_ORL.py and DSCN_L2_ORL.py I am getting an accuracy of only 15%. 
While when I run from the original authors in tensorflow I get 84% accuracy. 

Can you comment on why is this

**Comment by monster-xie** [comment]
I have same problem. is pre-train too short?

**Comment by 9310gaurav** [comment]
@kowshikthopalli  @monster-xie Hi guys! I'm afraid to inform you that while implementing the method I found it is very sensitive to the number of epochs you pre-train it to. And the epoch at which you get the best results also is very sensitive to the random seed and the intitialization. I was not able to complete the task of reproducing the tensorflow results even though I'm pretty sure my PyTorch implementation is an exact reproduction of the tensorflow code of the authors. If you guys find anything I've missed out, feel free to collaborate. Thanks!

**Comment by monster-xie** [comment]
the tensorflow pre-training is very long time,from cost=232592.76315789 to correct cost .I guess the pytorch pre-trainned epoch 9000 is too short!

**Comment by 9310gaurav** [comment]
Maybe. If you are able to run my model for longer epochs let me know the results you get.

**Comment by sushruthn96** [comment]
Is this issue solved ?
